roles/azure_manage_networking_stack/tasks/create.yml

```diff
@@ -12,7 +34,7 @@
     azure_manage_resource_group_name: "{{ azure_manage_networking_stack_resource_group }}"
     azure_manage_resource_group_region: "{{ azure_manage_networking_stack_region }}"
     azure_manage_resource_group_tags: "{{ azure_manage_networking_stack_tags }}"
```

Comment:
```suggestion
    azure_manage_resource_group_tags: "{{ azure_manage_networking_stack_tags  | default(omit) }}"
```

Reply:
Did you have an error?
I see this is not used in all roles, thinking if it should be added to all

Reply:
I haven't run the test yet, but I don't think that tag is a mandatory field. At least in the azure_managed_resource_group role it is omitted by default.